web/react/utils/markdown.jsx

```diff
@@ -223,6 +223,14 @@ class MattermostMarkdownRenderer extends marked.Renderer {
         return `<table class="markdown__table"><thead>${header}</thead><tbody>${body}</tbody></table>`;
     }
 
+    listitem(text) {
+        const taskListReg = /^\[([ |xX])\] /;
+        const isTaskList = taskListReg.exec(text);
+        return isTaskList ?
```

Comment:
Can you change this to not use nested ternary statements? This should've been picked up by ESLint